packages/babel-helper-transform-fixture-test-runner/src/index.ts

```diff
@@ -79,12 +79,11 @@ function createContext() {
 function runCacheableScriptInTestContext(
   filename: string,
   srcFn: () => string,
-  // todo(flow->ts) was Context type, but it is missing
-  context: any,
+  context: vm.Context,
   moduleCache: any,
 ) {
-  // todo(flow->ts) improve types
-  let cached: any = cachedScripts.get(filename);
+  let cached: { code?: string; cachedData?: Buffer } =
+    cachedScripts.get(filename);
```

Comment:
Instead of here, can you move the type annotation to the `cachedScripts` definition? Something like `new QuickLRU<string, { code: string; cachedData?: Buffer }>()`.

Reply:
:+1: https://github.com/babel/babel/pull/13738/commits/10a57fe0e9ef9deafc9a19945f95ac426631c1f5